app/api/sponsors.py

```diff
@@ -41,13 +43,24 @@ class Meta:
                          type_='event')
 
 
-class SponsorList(ResourceList):
+class SponsorListPost(ResourceList):
     """
     List and create Sponsors
     """
     def before_post(self, args, kwargs, data):
         require_relationship(['event'], data)
+        if not has_access('is_coorganizer', event_id=data['event']):
+            raise ForbiddenException({'source': ''}, 'Co-organizer access is required.')
+
+    schema = SponsorSchema
```

Comment:
Should this be made `POST` only? Because we have `GET /v1/sponsors` from SponsorList class already, and the permissions for get are being checked with query method there.